README.md

```diff
@@ -237,7 +237,11 @@ prettier.format(source, {
   jsxBracketSameLine: false,
 
   // Which parser to use. Valid options are 'flow' and 'babylon'
-  parser: 'babylon'
+  parser: 'babylon',
+
+  // Whether to add a semicolon at the end of every line (semi: true),
+  // or only at the beginning of lines that may introduce ASI failures (semi: false)
+  semi: true
```

Comment:
I think it's easier to have options that are by default false, so that people can just add them on the CLI easier. How about "noSemi" and the CLI arg would be `--no-semi`? For some reason I just like the args themselves describing the change.